### ISSUE TITLE
JAWS doesn't handle grid correctly. "Not in a table" when using ctrl+alt+arrow

### ISSUE THREAD

**Issue by timotheeguerin** [FreedomScientific/VFO-standards-support#146]
### Summary

Unable to use `role="grid"` with JAWS table navigation.
When you focus on a grid and then try to use ctrl+alt+arrow to navigate table, JAWS just says `Not in a table.` 
A role=table cannot be used with selection so a grid has to be used.

### Expected result

JAWS should allow to navigate in grid in browse mode

### Actual result

JAWS doesn't actually detect the grid correctly and is unable to do the table navigation

### Example
Repro https://jsfiddle.net/8L17eyam/2/
This is mockup the first row is selected and you can't change the selection but the idea remains.

#### JAWS version and build number
JAWS 2018 1811.2 ILM

#### Operating System and version
Windows 10

#### Browser and version:
Chrome `Version 70.0.3538.110 (Official Build) (64-bit)`

**Comment by LJWatson** [comment]
This is expected behaviour for something with `role="grid"`. The role is intended to trigger applications mode (forms mode in Jaws), which causes keystrokes to be passed back through to the application/browser. When you use `role="grid"`, the expectation is that JavaScript will be used to provide the expected keyboard interaction/support.

Can you describe the problem with 'role="table"' that you mention, in more detail?
>A role=table cannot be used with selection so a grid has to be used.>

**Comment by timotheeguerin** [comment]
I see maybe I am misunderstanding the expected behaviors.

According to this doc: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role
```
If a table maintains a selection state, has two-dimensional navigation, or allows the user to rearrange cell order use grid or treegrid instead.
```
I do need rows to be selectable. Also when using role=table JAWS doesn't pass down the key press(Arrow) to change the active row.

**Comment by LJWatson** [comment]
If you need rows to be selectable then `grid` might well be what you need. If it is, then you'll need to provide all the keyboard (including screen reader) interaction yourself using JavaScript. This includes replacing standard table navigation commands for moving up/down through columns, and left/right through rows.

If the table simply needs to have sortable columns and/or rows, then it's generally much easier to use an HTML table (as your demo does under the hood) or to use the `table` role if you have to.

**Comment by timotheeguerin** [comment]
I see. It was that my grid only provide navigating between rows and not columns and making the navigation row+column seems a bit weird as it doesn't do anything for non screen reader user.
But I guess this is something I could change
